users/tests/tests.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from django.test import TestCase, RequestFactory
+
+from users.forms import RegistrationForm
+from users.views import register
+
+
+class RegistrationPageUnitTests(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+
+    def test_method_get(self):
+        request = self.factory.get(reverse('register'))
+        response = register(request)
+        self.assertEqual(response.status_code, 200)
+
+    def test_method_post(self):
+        request = self.factory.post(reverse('register'))
+        response = register(request)
+        self.assertEqual(response.status_code, 200)
+
+    def test_form_is_valid(self):
+        form_data = {'username': "testUser", 'password1': "pass1234", 'password2': "pass1234"}
+        form = RegistrationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+    def test_form_is_invalid(self):
+        form_data = {'username': "testUser", 'password1': "pass1234", 'password2': "pass123"}
+        form = RegistrationForm(data=form_data)
+        self.assertFalse(form.is_valid())
```

Comment:
This method belongs to a new class. `RegistrationFormUnitTests` is the name I would suggest for that class.